src/pages/en/guides/styling.md

```diff
@@ -268,368 +246,70 @@ Lastly, add it to your Astro page (or layout template):
-1. Avoid global media queries
-
-#### Suggestion #1: Split your app into Layout Components and Base Components
-
-While this guide will never be long enough to answer the question _"How should a page be laid out?"_ (that’s a [design problem!][cassie-evans-css]) there is a more specific question hiding within that we _can_ answer: _"Given a layout, how should components/styles be organized?"_ The answer is **don't bake layout into components.** Have layout components that control layout, and base components (buttons, cards, etc.) that don't control layout. _What does that mean?_ Let’s walk through an example so it’s more clear. Pretend we have a page that looks like this (numbers for different components):
-
-```
-|---------------|
-|       1       |
-|-------+-------|
-|   2   |   2   |
-|---+---|---+---|
-| 3 | 3 | 3 | 3 |
-|---+---+---+---|
-| 3 | 3 | 3 | 3 |
-|---+---+---+---|
-```
-
-The layout consists of a big, giant, full-width post at top, followed by two half-width posts below it. And below that, we want a bunch of smaller posts to fill out the rest of the page. For simplicity, we'll just call these `<BigPost>` (1), `<MediumPost>` (2), and `<SmallPost>` (3). We add them to our page like so:
-
-```astro
----
-// src/pages/index.astro
-
-import Nav from '../components/Nav.astro';
-import BigPost from '../components/BigPost.astro';
-import Grid from '../components/Grid.astro';
-import MediumPosts from '../components/MediumPosts.astro';
-import SmallPosts from '../components/SmallPosts.astro';
-import Footer from '../components/Footer.astro';
----
-<html>
-  <body>
-    <Nav />
-
-    <Grid>
-      <BigPost />
-      <MediumPosts />
-      <SmallPosts />
-    </Grid>
-
-    <Footer />
-  </body>
-</html>
-```
-
-This _looks_ clean, but looks can be deceiving. At first glance, we may think that `<Grid>` is controlling the layout, but that’s an illusion. We actually have `<BigPost>` handling its own width, `<MediumPosts>` loading 2 components and controlling its width, and `<SmallPosts>` loading 4+ components and controlling its width. In total, including `<Grid>`, that means **4 components** are all fighting over the same layout. Remove one post from `<MediumPosts>`, the layout breaks. Edit `<BigPost>`, the layout breaks. Edit `<Grid>`, the layout breaks. If you think about it, none of these components are truly reusable—they might as well just be one big file.
+- **React** / **Preact**: `import Styles from './styles.module.scss'`;
+- **Vue**: `<style lang="scss">`
+- **Svelte**: `<style lang="scss">`
 
-This is actually the **Global CSS Problem** in disguise—multiple components fight over how they all lay out together, without layout being one, central responsibility (kinda like global CSS)! Now that we identified the problem, one way to fix this is to hoist the entire layout to the top level, and load all components there, too:
+Additionally, [PostCSS](#-postcss) is supported, but the setup is [slightly different](#-postcss).
```

Comment:
Broken link?

Reply:
Ah, I reorganized this page, so yes, an internal link based on headers would have changed.